sdk/test-utils/test-utils/src/tracing/azureTracing.ts

```diff
@@ -0,0 +1,127 @@
+import { OperationTracingOptions, useInstrumenter } from "@azure/core-tracing";
```

Comment:
I wonder why the linter passed without a valid header?

Reply:
Yeah not sure why linter is not working with this file